kadai-adapter-camunda-spring-boot-test/src/test/java/io/kadai/adapter/integration/HttpHeaderProviderWithCsrfToken.java

```diff
@@ -0,0 +1,35 @@
+package io.kadai.adapter.integration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.kadai.adapter.systemconnector.camunda.api.impl.HttpHeaderProvider;
+import io.kadai.adapter.test.KadaiAdapterTestApplication;
+import io.kadai.common.test.security.JaasExtension;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.reactive.AutoConfigureWebTestClient;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.http.HttpHeaders;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+
+@SpringBootTest(
+    classes = KadaiAdapterTestApplication.class,
+    webEnvironment = WebEnvironment.DEFINED_PORT)
+@AutoConfigureWebTestClient
+@ExtendWith(JaasExtension.class)
+@ContextConfiguration
+@TestPropertySource(properties = {"kadai.adapter.xsrf.token=KAD_UNIQUE_TOKEN_123"})
+class HttpHeaderProviderWithCsrfToken {
```

Comment:
Test class usually always have TestXXX oder XXXTest in the name...did we just miss this here or is there a specific reason?